scripts/Tools/preview_run

```diff
@@ -51,38 +50,7 @@ def _main_func(description):
     logging.disable(logging.INFO)
 
     with Case(caseroot, read_only=False) as case:
-        print("CASE INFO:")
-        print("  nodes: {}".format(case.num_nodes))
-        print("  total tasks: {}".format(case.total_tasks))
-        print("  tasks per node: {}".format(case.tasks_per_node))
-        print("  thread count: {}".format(case.thread_count))
-        print("")
-
-        print("BATCH INFO:")
-        if not job:
-            job = case.get_first_job()
-        set_logger_indent("      ")
-        job_id_to_cmd = case.submit_jobs(dry_run=True, job=job)
-        env_batch = case.get_env('batch')
-        for job_id, cmd in job_id_to_cmd:
-            print("  FOR JOB: {}".format(job_id))
-            print("    ENV:")
-            case.load_env(job=job_id, reset=True, verbose=True)
```

Comment:
Can we change this to use the --verbose flag instead of vebose=True?  The --verbose flag is already defined to preview_run in standard_script_logging_options.

Reply:
I harded it to false to prevent printing the environment variables twice, the first time was without any formatting and I had assumed the user would always want to see them.

If my assumption was wrong I can fix the double printing and link it to the verbose flag.